src/value_serializer.rs

```diff
@@ -0,0 +1,109 @@
+use crate::Context;
+use crate::HandleScope;
+use crate::Isolate;
+use crate::Local;
+
+use crate::support::MaybeBool;
+
+use crate::data::ArrayBuffer;
+use crate::data::Value;
+
+use std::ffi::c_void;
+use std::mem::MaybeUninit;
+
+/// Instances of this class can be passed
```

Comment:
1. I'm not sure exactly what you mean here - maybe "instances of this class can be passed by value"?
2. Triple slash comments are used for doc comments. That means that if we were to publish the crate like this
    the words "Instances of this class..." would be visible [here](https://docs.rs/rusty_v8/0.8.1/rusty_v8/#functions).
    For code-only comments, use `//` instead, like you've done on the line below.